src/molecules/Footer.tsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react'
+import Layout, { Box, Composition } from 'atomic-layout'
+import styled from 'styled-components'
+import { Logo } from '../atoms/Logo'
+import { Text } from '../atoms/Text'
+import {
+  IoLogoFacebook,
+  IoLogoTwitter,
+  IoLogoInstagram,
+  IoLogoVk
+} from 'react-icons/io'
+
+const templateMobile = `
+	contacts
+	products
+	links
+	newsletter
+	legal
+`
+
+const templateMd = `
+	contacts products links
+	newsletter newsletter newsletter
+	legal legal legal
+`
+const templateDesktop = `
+contacts products links newsletter
+legal legal legal legal
+`
+
+const StyledFooter = styled.div`
+  background-color: ${({ theme }) => theme.colors.greyDim};
+  color: ${({ theme }) => theme.colors.greyLight};
+  margin: 0;
+
+  @media (max-width: ${Layout.breakpoints.sm.maxWidth}) {
+    text-align: center;
+  }
+`
+
+const StyledContainerCopyright = styled.div`
+  border-top: 1px solid #ddd;
+`
+
+const StyledImage = styled.img`
+  height: auto;
+  max-width: 100%;
+  margin: 14px 0;
+  display: flex;
+  justify-self: center;
+
+  @media (min-width: ${Layout.breakpoints.md.minWidth}) {
+    justify-self: end;
+  }
+`
+
+const StyledContact = styled.div`
+  color: ${({ theme }) => theme.colors.blackLight};
+`
+
+const StyledP = styled.p`
+  margin: 0;
+`
+
+const StyledLink = styled.a`
+  color: ${({ theme }) => theme.colors.greyLight};
+`
+
+const StyledUl = styled.ul`
+  margin: 0;
+  padding: 0;
+  list-style: none;
+`
+
+const StyledList = styled.li`
+  margin-bottom: 10px;
```

Comment:
This will have an extra 10px margin on the last list item (`li`). Do you need that?